apps/src/templates/ChallengeDialog.story.jsx

```diff
@@ -3,158 +3,135 @@ import CodeWritten from './feedback/CodeWritten';
-    {
-      name: 'Passed Dialog with a bird',
-      description: 'Too many blocks',
-      story: () => (
-        <div style={wrapperStyle}>
-          <ChallengeDialog
-            hideBackdrop
-            avatar="/blockly/media/skins/birds/win_avatar.png"
-            title="You did it!"
-            primaryButtonLabel="Continue"
-            cancelButtonLabel="Try again"
-            showPuzzleRatingButtons
-            text="However, you could've done it with only N blocks. Can you make your program even better?"
-          >
-            <CodeWritten numLinesWritten={9} useChallengeStyles>
-              <GeneratedCode
-                message="Here's your code:"
-                code="console.log('U is for you and me!');"
-              />
-            </CodeWritten>
-          </ChallengeDialog>
-        </div>
-      )
-    },
-    {
-      name: 'Perfect Dialog',
-      description: 'Perfect completion',
-      story: () => (
-        <div style={wrapperStyle}>
-          <ChallengeDialog
-            hideBackdrop
-            avatar="/blockly/media/skins/harvester/win_avatar.png"
-            complete
-            title="Challenge Complete!"
-            primaryButtonLabel="Continue"
-            cancelButtonLabel="Replay"
-            showPuzzleRatingButtons
-          >
-            <CodeWritten numLinesWritten={9} useChallengeStyles>
-              <GeneratedCode
-                message="Here's your code:"
-                code="console.log('N is for anywhere and anytime at all');"
-              />
-            </CodeWritten>
-          </ChallengeDialog>
-        </div>
-      )
-    },
-    {
-      name: 'Perfect Dialog without puzzle ratings',
-      description: 'Perfect completion',
-      story: () => (
-        <div style={wrapperStyle}>
-          <ChallengeDialog
-            hideBackdrop
-            avatar="/blockly/media/skins/studio/win_avatar.png"
-            complete
-            title="Challenge Complete!"
-            primaryButtonLabel="Continue"
-            cancelButtonLabel="Replay"
-          >
-            <CodeWritten numLinesWritten={9} useChallengeStyles>
-              <GeneratedCode
-                message="Here's your code:"
-                code="console.log('N is for anywhere and anytime at all');"
-              />
-            </CodeWritten>
-          </ChallengeDialog>
-        </div>
-      )
-    }
-  ]);
+// TEMPLATE
+
+const Template = args => (
+  <div style={wrapperStyle}>
+    <ChallengeDialog {...args} />
+  </div>
+);
+
+// STORIES
+
+export const Starting_Dialog = Template.bind({});
+Starting_Dialog.args = {
+  hideBackdrop: true,
+  avatar: '/blockly/media/skins/harvester/static_avatar.png',
+  cancelButtonLabel: 'Skip for now',
+  primaryButtonLabel: "I'm Ready!",
+  text:
+    'Challenge Puzzles are lessons designed to push your skills to a new level.',
+  title: 'Challenge Puzzle!',
+  isIntro: true
+};
+
+export const Starting_Dialog_with_Large_Avatar = Template.bind({});
```

Comment:
Nit: are underscores a norm?

Reply:
Oh hmm, I thought I had seen underscores in an example but looking again I may have been looking at something else. I'll remove to be consistent